docs/changes.rst

```diff
@@ -31,6 +31,20 @@ Released: not yet
   surrounded by verbose test (if p.parse.verbose:). See pywbemcli issue
   #395,
 
+* Fixes bugs in SubscriptionManager where 1. host_name  of
+  CIM_ListenerDestination and CIM_Filter objects inconsistently used
+  in building the string in the Name property and building the regex to
+  find the string. Starting a subscription manager against a server with owned
+  subscriptions, etc. did not restore them as owned, 2. add_filter(),
+  add_destination() and add_subscriptions() did not create a complete path for
+  the instance to be added to be added. While this is correct for the
+  CreateInstance(), that path is used to determine if the instance already
+  exists a few lines further down in the code. 3. The client host fqdn was being
+  used as the instance SystemName property when the local instances were built.
+  WBEM servers consistently overwrite this property if is is incorrect or not
+  defined. The SystemName returned from the add_operations is now used to
+  create new paths for the add_...() methods.  (issue #701).
+
```

Comment:
Another fix is that the CIM instances for filters, subscriptions and listener destinations were created in the default namespace of the connection of the server that was registered, and are now correctly created in the Interop namespace regardless of the default namespace.

Reply:
Since the path was attached before the CreateInstance was called, we are covered but I added namespace to the CreateInstance calls.

DONE

Reply:
ok. You are right, it uses the namespace from the path if present.